deploy/infra/groups/data.bicep

```diff
@@ -147,7 +148,7 @@ module synapseIdentityForStorageAccess '../modules/storage-role-assignment.bicep
   ]
 }]
 
-module postgresqlServer '../modules/postgres.single.svc.bicep' = {
+module postgresqlServer '../modules/postgres.single.svc.bicep' = if(deployPgSQL) {
```

Comment:
@sushilkm  Should we move all the postgres related resources and module in one bicep under "groups" folder say, postgres.bicep and then apply the conditional deployment once to that bicep module instead of multiple times against each resource and module?